src/MCPServer/share/postinstSharedWithDev

```diff
@@ -7,17 +7,10 @@ else
   adduser --uid 333 --group --system --home /var/lib/archivematica/ archivematica
 fi
 
-sudo usermod -a -G audio archivematica
-sudo chown -R archivematica:archivematica "/var/archivematica/"
 sudo chown -R archivematica:archivematica "/var/archivematica/sharedDirectory/"
-sudo chmod -R g+s "/var/archivematica/"
-sudo chmod -R 664 "/var/archivematica/sharedDirectory" 
-sudo find -L /var/archivematica/ -type d  | sudo xargs -IF chmod u+rwx,g+rwxt,o-rwx F
-
-logdir=/var/log/archivematica/MCPServer
-sudo mkdir -p $logdir
-sudo chown -R archivematica:archivematica $logdir
-sudo chmod -R g+s $logdir
+sudo chmod -R g+s "/var/archivematica/sharedDirectory"
+sudo chmod -R 664 "/var/archivematica/sharedDirectory"
```

Comment:
These should also have a trailing / if we want to affect the directory not the (possible) symlink.